Background.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alina Pervez</title>
+    <link rel="stylesheet" href="style.css">
+ 
+</head>
+<body>
+    <div id="main">
+        
+        <div class="sidebar">
```

Comment:
consider using semantic tags (instead of divs) here and in other files to make your HTML structure more accessible: https://www.w3schools.com/html/html5_semantic_elements.asp